neuromation/api/parser.py

```diff
@@ -23,26 +23,50 @@ class Volume:
     read_only: bool = False
 
 
+@dataclass(frozen=True)
+class DiskVolume:
+    disk_uri: URL
+    container_path: str
+    read_only: bool = False
+
+
+@dataclass(frozen=True)
+class VolumeParseResult:
+    volumes: List[Volume]
+    secret_files: List[SecretFile]
+    disk_volumes: List[DiskVolume]
+
+
+@dataclass(frozen=True)
+class EnvParseResult:
+    env: Dict[str, str]
```

Comment:
`VolumeParseResult` use the plural form for fields: `volumes`, `secret_files` etc.
What is the reason for the single form (`env`) in `EnvParseResult`?

Reply:
By `env` I mean environment,  single form. I used plurals because `VolumeParseResult` contains `List` objects, but here is a single `Dict`. That was my point, but I can change to plurals if you think that way it will look cleaner.

Reply:
Got you. Makes sense